docs/tutorials/control/air-quality-fleet.md

```diff
@@ -797,28 +749,247 @@ You can find all the code in the [GitHub repo for this tutorial](https://github.
 
    ![Air quality dashboard in a web browser with PM2.5 readings from three different sensor machines displayed.](/tutorials/air-quality-fleet/three-sensor-dash.png)
 
-   Great work.
-   You've learned how to configure a fleet of machines, sync their data to one place, and pull that data into a custom dashboard using TypeScript.
+Great work.
+You've learned how to configure a machine and you can view its data in a custom TypeScript dashboard.
+
+### Deploy the app to Viam apps
+
+Let's deploy this dashboard so you don't have to run it locally.
```

Comment:
As said elsewhere, I think we use "deploy" to mean a lot of different things, so it'd be good to be more specific here. Without digging, not sure exactly what it should say but maybe:
```suggestion
Let's deploy this dashboard as a Viam-hosted app so you don't have to run it locally.
```